### PR TITLE
[HPRO-662] Add Oct COPE survey details to the Participant Work Queue and the Participant Summary pages

### PR DESCRIPTION
**Pull request by shyamnaru**

### CODE REVIEW

**Review by stephenyeargin — APPROVED** [review]

**Review by jt2k — APPROVED** [review]
Looks good. We should probably wait until we get confirmation that this is going to be ready in RDR prod with correct data before merging.

**Pull request merged**
